base/actions/actions.py

```diff
@@ -577,11 +577,24 @@ def get_healthcheck_data(self, tracker: Tracker, risk: Text) -> Dict[Text, Any]:
             "exposure": self.YES_NO_MAYBE_MAPPING[tracker.get_slot("exposure")],
             "tracing": self.YES_NO_MAPPING[tracker.get_slot("tracing")],
             "risk": risk,
-            "location": self.fix_location_format(tracker.get_slot("location_coords")),
-            "city_location": self.fix_location_format(
-                tracker.get_slot("city_location_coords")
-            ),
         }
+        if self.AGE_MAPPING[tracker.get_slot("age")] != "<18":
+            data.update(
```

Comment:
this could be data["location"] = self.fix_loc...

Reply:
For some reason it returns location as a list that way.
EX: {'location': ['+03.4-001.2/']}
so I had to do it this way.

Reply:
Hmm that doesn't make sense really, trying to pull the branch to see

Reply:
Cool. {'deduplication_id': '333333', 'msisdn': '+default', 'source': 'WhatsApp', 'province': 'ZA-WC', 'city': 'Cape Town, South Africa', 'age': '18-40', 'gender': 'not_say', 'cough': True, 'fever': False, 'sweat': True, 'weight': True, 'exposure': 'not_sure', 'tracing': True, 'risk': 'moderate', 'location': ('+03.4-001.2/',), 'city_location': ('+01.2-003.4/',)}


So: 
AssertionError: assert {'age': '18-4...h': True, ...} == {'age': '18-4...h': True, ...}
E         Omitting 13 identical items, use -vv to show
E         Differing items:
E         {'location': ['+03.4-001.2/']} != {'location': '+03.4-001.2/'}
E         {'city_location': ['+01.2-003.4/']} != {'city_location': '+01.2-003.4/'}

Payload and test:
```
```